src/main/webapp/script.js

```diff
@@ -327,10 +327,12 @@ function setupNavBarProfileSection() {
     .then((response) => response.json())
     .then((loginResponse) => {
       const profileLinks = $('#profileLinks');
-
+      const logLinks = $('#logLinks');
```

Comment:
what happens to the logLinks button if the user is not logged in? does it still appear?

Reply:
The logout and profile button will not show

Reply:
Sounds good!